### ISSUE TITLE
make sure it gives proper errors when run on whitespace or comment-only strings

### ISSUE THREAD

**Issue by pgbovine** [seankross/mario#39]
e.g.,:
- empty string
- a string of whitespaces
- only comments like "# just a comment here" <-- it currently displays something strange
- a legal R program but one that doesn't contain pipes or data frames

**Comment by pgbovine** [comment]
e.g.,

```
$ node run-backend.js r '#just a comment   '
{
  "code": "#just a comment   ",
  "trace": [
    {
      "type": "error",
      "code_step": "NA",
      "mapping": {
        "message": {
          "header": "attempt to select less than one element in integerOneIndex",
          "body": [],
          "footer": []
        }
      },
      "data_frame": "NA"
    }
  ]
}
```

```
$ node run-backend.js r '   '
{
  "code": "   ",
  "trace": [
    {
      "type": "error",
      "code_step": "NA",
      "mapping": {
        "message": {
          "header": "attempt to select less than one element in integerOneIndex",
          "body": [],
          "footer": []
        }
      },
      "data_frame": "NA"
    }
  ]
}
```

**Comment by seankross** [comment]
Fixed in 18edf68
